app/javascript/packs/articlePage.jsx

```diff
@@ -178,6 +178,16 @@ if (profilePreviewTrigger?.dataset.initialized !== 'true') {
   initializeDropdown({
     triggerElementId: 'profile-preview-trigger',
     dropdownContentId: 'profile-preview-content',
+    onOpen: () => {
+      document
```

Comment:
**(refactor, nonblocking)**: What about assigning `document.getElementById('profile-preview-content')` to a variable like in the `commentDropdown` instead of querying it over in the `onOpen` and `onClose`

Reply:
Refactor (potentially-blocking): If you stick with `document.getElementById`, might as well set `dropDownContentId` as a variable and use it in all the spots `'profile-preview-content'` in the scope of this function/code block.